tests/codeception/functional/TemplateCest.php

```diff
@@ -54,6 +54,25 @@ public function testEditTemplate(FunctionalTester $I)
         $I->see('Edited First Template');
     }
 
+    public function testGetTemplateContentApi(FunctionalTester $I)
+    {
+        $I->wantTo('Get template content via JSON endpoint and verify response structure');
+
+        $space = $I->loginBySpaceUserGroup(Space::USERGROUP_ADMIN);
+        $I->enableModule($space->guid, 'wiki');
+
+        $template = WikiTemplate::find()
+            ->where(['title' => 'Edited First Template'])
+            ->one();
+
+        $I->amOnSpace($space->guid, '/wiki/template/get-template-content', ['id' => $template->id]);
+        $I->seeResponseCodeIs(200);
+
+        $I->seeInSource('"success":true');  
+        $I->seeInSource('"content":"Content of First Template"');
```

Comment:
Please try to use some markdown code in here to assert, that markdown and not HTML is returned.